docs/source/json-api/http-status-codes.rst

```diff
@@ -0,0 +1,125 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+HTTP Status Codes
```

Comment:
I’m not sure moving this to a completely separate section is a good idea. Based on the feedback I’ve seen for the existing docs, a fair amount of people miss those sections and end up assuming this information does not exist at all.

Reply:
I think sections get missed because the doc (and resulting navigation) is so long which was my initial reason for splitting this up to a separate sub-section.

Another thing I would like to do in addition or in place of breaking out the HTTP Status Codes subsection is reduce the section menu's navigation depth. But my attempts to do something like the following haven't worked.

```
.. content:: 
  :depth: 2
```

Which would ideally reduce this:
![image](https://user-images.githubusercontent.com/59614480/100250612-d9ca6d80-2f0b-11eb-9164-ac1b02d2e0e7.png)

To this, which is much easier to read IMO:
![image](https://user-images.githubusercontent.com/59614480/100251095-607f4a80-2f0c-11eb-9c0d-b0da1b2ec522.png)

Reply:
@anthonylusardi-da Think if you wanna do that you need to use `maxdepth:1` for `toctree` as per [this doc](https://www.sphinx-doc.org/en/1.5/markup/toctree.html)

Reply:
`toctree` is actually for the left hand overall nav menu, not the right hand section menu. I'm not sure what's wrong here but I assume our build system might just be ignoring `content`